programs/lz4cli.c

```diff
@@ -49,7 +49,7 @@
 #  pragma warning(disable : 4127)      /* disable: C4127: conditional expression is constant */
 #endif
 
-#define _POSIX_SOURCE 1        /* for fileno() within <stdio.h> on unix */
+#define _XOPEN_VERSION 600 /* POSIX.2001, for fileno() within <stdio.h> on unix */
```

Comment:
Sorry, I meant a link to the authoritative source.
For example : http://man7.org/linux/man-pages/man7/feature_test_macros.7.html

It can be quite useful to add such link into code comments, so that future reviewers are aware of the choice's basis.